### DISCUSSION TITLE
Missing definition

### DISCUSSION THREAD

**Discussion by DesertDwellerEv** [zachowj/node-red-contrib-home-assistant-websocket#318] [Cookbook]
Sample for the Entities Page includes sensor.holiday_lights_demo_current
However, I can't seem to find any references to it anywhere else.
Did I miss something?

**Comment by zachowj** [comment]
It should be the sensor created by the entity node `current demo`, bottom right corner.

**Reply by DesertDwellerEv** [reply]
Is shows the following:
![image](https://user-images.githubusercontent.com/53842308/103430928-154cfd00-4b86-11eb-90e3-a26c8747540c.png)
Should it be holiday_lights_demo_current instead of Holiday Lights Demo Current?
I checked and have hass-node-red installed thru HACS
Am I missing anything else?

**Reply by zachowj** [reply]
Using the [custom integration](https://github.com/zachowj/hass-node-red) that node creates the sensor in HA. Which when HA converts the name to the entity id slug it becomes `sensor.holiday_lights_demo_current`

**Reply by DesertDwellerEv** [reply]
Looks like I missed the step of adding Node Red Integration.
Sensor was created, now I get the following in **_populate HA dropdown with presets_** node is called:
Call-service API error.  Error Message: length of value must be at least 1 for dictionary value @ data['options']
Is that because I don't have any presets?
Thanks again for your help. Have a safe and Happy New Year!

**Reply by zachowj** [reply]
> Is that because I don't have any presets?

Yes, that sounds right.
